agent/engine/dependencygraph/graph.go

```diff
@@ -219,6 +220,10 @@ func verifyContainerOrderingStatusResolvable(target *apicontainer.Container, exi
 		if !ok {
 			return fmt.Errorf("dependency graph: container ordering dependency [%v] for target [%v] does not exist.", dependencyContainer, target)
 		}
+		if !hasDependencyTimedOut(dependencyContainer, dependency.Condition) {
```

Comment:
If you take my suggestion to make the function return true when there is a time out, you can get rid of the `!`